components/requests/RequestHeader.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  Box,
+  Flex,
+  HStack,
+  Text,
+  Menu,
+  MenuList,
+  MenuItem,
+  MenuButton,
+  Button,
+} from '@chakra-ui/react'; // Chakra UI
+import { ChevronLeftIcon, ChevronDownIcon } from '@chakra-ui/icons'; // Chakra UI icon
+import Link from 'next/link'; // Link
+import RequestStatusBadge from '@components/internal/RequestStatusBadge'; // Request status badge
+import ApproveRequestModal from '@components/requests/modals/ApproveRequestModal'; // Approve button + modal
+import RejectRequestModal from '@components/requests/modals/RejectRequestModal'; // Reject button + modal
+import CompleteRequestModal from '@components/requests/modals/CompleteRequestModal'; // Mark as complete button + modal
+
+type RequestHeaderProps = {
+  readonly isRenewal: boolean;
+  readonly applicationStatus: // TODO: Change this to the enum that we add
+  'COMPLETED' | 'INPROGRESS' | 'PENDING' | 'REJECTED' | 'EXPIRING' | 'EXPIRED' | 'ACTIVE';
+  readonly createdAt: string;
```

Comment:
Note: Not sure of this yet, but if Apollo client parses the date scalar and converts it to a date object, we want to change this prop to be a `Date` type